### PR TITLE
chore: update subscription npm version

### PR DESCRIPTION
**Pull request by xiaolang124**
1.Update the subscription version to the newest.

This is a part of removing cli deprecated work.
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15066253/

In the future pr, we will remove Azure `getAccountCredentialAsync` method.

### CODE REVIEW

**Comment by xiaolang124** [comment]
e2e test: https://github.com/OfficeDev/TeamsFx/actions/runs/2815229319

Failed cases are same with daily pipeline.

File: packages/cli/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   },
   "aiKey": "1c56be97-bb74-42cf-b04b-8f1aabf04592",
   "devDependencies": {
-    "@azure/arm-subscriptions": "^3.0.0",
+    "@azure/arm-subscriptions": "^5.0.0",
```
Comment (swatDong) [inline packages/cli/package.json:45]: Remove this since it's already in `dependencies`?

Comment (xiaolang124) [reply packages/cli/package.json]: Done.

File: packages/fx-core/src/plugins/solution/fx-solution/utils/util.ts [hunk 2]
```diff
@@ -17,6 +17,16 @@ import {
 import { SubscriptionClient } from "@azure/arm-subscriptions";
 import { SolutionTelemetryComponentName, SolutionTelemetryProperty } from "../constants";
 import { TokenCredential } from "@azure/core-auth";
+import { BuiltInFeaturePluginNames } from "../v3/constants";
```
Comment (swatDong) [inline packages/fx-core/src/plugins/solution/fx-solution/utils/util.ts:20]: What are these import changes for?

**Review by swatDong — APPROVED** [review]

**Pull request closed without merging**